ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -151,10 +145,8 @@ const AssetListControlBar = ({
   }, [tokenNetworkFilter, allNetworks]);
 
   const networksToDisplay = useMemo(() => {
-    return isGlobalNetworkSelectorRemoved
-      ? enabledNetworksByNamespace
-      : tokenNetworkFilter;
-  }, [tokenNetworkFilter, enabledNetworksByNamespace]);
+    return enabledNetworksByNamespace;
```

Comment:
small: with a single return here looks this can be simplified no?

const networksToDisplay = enabledNetworksByNamespace

or just the other variable directly?